### PR TITLE
{storage-blob-preview} Storage blob features with api version 2019-12-12

### PR DESCRIPTION
**Pull request by Juliehzl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

This is new extension for storage blob track2.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
storage-blob-preview

File: src/index.json [hunk 1]
```diff
@@ -7701,6 +7701,7 @@
                 "filename": "spring_cloud-0.2.2-py2.py3-none-any.whl",
                 "metadata": {
                     "azext.isPreview": true,
+                    "azext.maxCliCoreVersion": "2.8.0",
```
Comment (haroldrandom) [inline src/index.json:7704]: Should revert it.

File: src/index.json [hunk 2]
```diff
@@ -7913,6 +7914,7 @@
                 "filename": "spring_cloud-0.3.0-py2.py3-none-any.whl",
                 "metadata": {
                     "azext.isPreview": true,
+                    "azext.maxCliCoreVersion": "2.8.0",
```
Comment (haroldrandom) [inline src/index.json:7917]: Should revert it.

File: src/index.json [hunk 3]
```diff
@@ -7958,6 +7960,7 @@
                 "filename": "spring_cloud-0.3.1-py2.py3-none-any.whl",
                 "metadata": {
                     "azext.isPreview": true,
+                    "azext.maxCliCoreVersion": "2.8.0",
```
Comment (haroldrandom) [inline src/index.json:7963]: Should revert it.

File: src/index.json [hunk 4]
```diff
@@ -8003,6 +8006,7 @@
                 "filename": "spring_cloud-0.4.0-py2.py3-none-any.whl",
                 "metadata": {
                     "azext.isPreview": true,
+                    "azext.maxCliCoreVersion": "2.8.0",
```
Comment (haroldrandom) [inline src/index.json:8009]: Should revert it.

File: src/index.json [hunk 5]
```diff
@@ -8048,6 +8052,7 @@
                 "filename": "spring_cloud-0.5.0-py2.py3-none-any.whl",
                 "metadata": {
                     "azext.isPreview": true,
+                    "azext.maxCliCoreVersion": "2.8.0",
```
Comment (haroldrandom) [inline src/index.json:8055]: Should revert it.

File: src/index.json [hunk 6]
```diff
@@ -8093,6 +8098,7 @@
                 "filename": "spring_cloud-0.5.1-py2.py3-none-any.whl",
                 "metadata": {
                     "azext.isPreview": true,
+                    "azext.maxCliCoreVersion": "2.8.0",
```
Comment (haroldrandom) [inline src/index.json:8101]: Should revert it.

**Review by haroldrandom — APPROVED** [review]

**Pull request merged**